src/Listener/SaveTrelloIdToDatabase.php

```diff
@@ -144,4 +146,17 @@ private function attachLabelsToCardBasedOnForumTags(Discussion $discussion, Card
             }
         }
     }
+
+    private function attachMembersToCard(Card $card, $members)
+    {
+        $client = resolve(Client::class);
```

Comment:
No need to resolve the client again here, it's available already `$this->client`